File: JavaProjects/FactoryMethodSolucao/src/boletobancario/bancodobrasil/BancoDoBrasilBoleto60Dias.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package boletobancario.bancodobrasil;

import boletobancario.Boleto;

/**
 *
 * @author jeffe
 */
public class BancoDoBrasilBoleto60Dias extends Boleto {

    public BancoDoBrasilBoleto60Dias(double valor) {
        this.valor=valor;
        this.juros = 0.1;
        this.desconto=0;
        this.multa=0.15;
    }
}

File: CsharpProjects/FactoryMethodSolucao/FactoryMethodSolucao/boletobancario/Bancodobrasil/BancoDoBrasilBoleto60Dias.cs
using System;
using System.Collections.Generic;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using FactoryMethodSolucao.boletobancario;

namespace FactoryMethodSolucao.boletobancario.Bancodobrasil
{
    public class BancoDoBrasilBoleto60Dias : Boleto
    {
        public BancoDoBrasilBoleto60Dias(double valor)
        {
            this.valor = valor;
            juros = 0.1;
            desconto = 0;
            multa = 0.15;
        }
    }
}
